src/webui/scripts/start.js

```diff
@@ -54,35 +55,51 @@ if (process.env.HOST) {
   console.log(
     `If this was unintentional, check that you haven't mistakenly set it in your shell.`
   );
-  console.log(`Learn more here: ${chalk.yellow('http://bit.ly/2mwWSwH')}`);
+  console.log(
+    `Learn more here: ${chalk.yellow('https://bit.ly/CRA-advanced-config')}`
+  );
   console.log();
```

Comment:
empty content

Reply:
Thanks for reviewing! I had delete this line.